index.loopback.html

```diff
@@ -165,21 +165,21 @@
   <div class="container">
       <div class="row text-center">
           <div class="col-md-3">
-              <img src="https://v4.loopback.io/img/global/loopback-mark-frame-white.svg" width=40px/>
+              <img src="https://loopback.io/img/global/loopback-mark-frame-white.svg" width=40px/>
           </div>
           <div class="col-md-3">
               <p class="label text-uppercase">Product</p>
               <p class="regular">
-                  <a href="http://loopback.io/doc/en/lb4/Getting-started.html">Getting Started</a>
+                  <a href="https://loopback.io/doc/en/lb4/Getting-started.html">Getting Started</a>
               </p>
               <p class="regular">
-                  <a href="http://loopback.io/doc/en/lb4/Examples-and-tutorials.html">Tutorials</a>
+                  <a href="https://loopback.io/doc/en/lb4/Examples-and-tutorials.html">Tutorials</a>
```

Comment:
This link no longer exist. At some point, we split the examples and tutorials into 2 pages, i.e. 
https://loopback.io/doc/en/lb4/Tutorials.html
https://loopback.io/doc/en/lb4/Examples.html